packages/frontend-2/lib/viewer/composables/savedViews/ui.ts

```diff
@@ -14,6 +14,9 @@ import { isUngroupedGroup } from '@speckle/shared/dist/esm/saved-views/index.js'
 const isDraggableView = (view: unknown): view is UseDraggableView_SavedViewFragment =>
   isObjectLike(view) && has(view, 'id') && has(view, 'permissions.canUpdate')
 
+// Shared state to track which view is currently being dragged
+const currentlyDraggingViewId = ref<string | null>(null)
```

Comment:
JS module root scope refs/composables are a no go https://vuejs.org/guide/scaling-up/ssr.html#cross-request-state-pollution

if a global/singleton state is needed, useState/useScopedState must be used

but more importantly - why do we need this in the first place? from drop target composables you can always get the currently dragged view through eventy.dataTransfer.getData(). am i missing something?